carbon-design-system/carbon
# fix(multiselect): listbox hidden when closed

Closes #11341 


#### Changelog

- make listbox menu hidden in the a11y tree when multiselect is closed

#### Testing / Reviewing

- go to multiselect storybook and run the a11y checker on the closed menu
    - this a11y warning should no longer appear
    <img width="919" alt="Screen Shot 2022-08-11 at 4 28 39 PM" src="https://user-images.githubusercontent.com/32556167/184245982-bbe287c7-257e-4d8b-a9fc-0c7e986b1b7e.png">

- ensure all tests are passing and component is working as expected
